index.bs

```diff
@@ -755,21 +807,40 @@ if there exists an |entry| of |list| that is an [=ordered map=], and where
 </div>
 
 <div algorithm>
-To <dfn for="SanitizerConfig">remove</dfn> an |item| from a |list| that is an
-[=ordered map=], [=list/remove=] all |entry| from |list|
-where |item|["name"] [=equals=] |entry|["name"] and
-|item|["namespace"] [=equals=] |entry|["namespace"].
+To <dfn for="SanitizerConfig">remove</dfn> an |item| from the key |key| in a
+|configuration|:
+
+1. If |configuration|[|key|] does not [=map/exist=], return.
+1. Let |list| be |configuration|[|key|].
+1. [=Assert=]: |list| is an [=ordered map=].
+1. [=list/Remove=] all |entry| from |list| where |item|["name"]
+    [=equals=] |entry|["name"] and
+    |item|["namespace"] [=equals=] |entry|["namespace"].
+
 </div>
 
 <div algorithm>
-To <dfn for="SanitizerConfig">add</dfn> a |name| to a |list|, where |name| is
-[=canonicalize a sanitizer name|canonicalized=] and |list| is an [=ordered map=]:
-
+To <dfn for="SanitizerConfig">add</dfn> a |name| to the key |key| in a
+|configuration|, where |name| is
+[=canonicalize a sanitizer name|canonicalized=] and
+|configuration| is a {{SanitizerConfig}}:
+
+1. If |configuration|[|key|] does not [=map/exist=], set
+    |configuration|[|key|] to &laquo;[]&raquo;.
+1. Let |list| be |configuration|[|key|].
 1. If |list| [=SanitizerConfig/contains=] |name|, then return.
 1. [=list/Append=] |name| to |list|.
 
 </div>
 
+<div algorithm>
+To determine the <dfn for="SanitizerConfig">size</dfn> of a [=/list=] |list|:
+
+1. If |list| does not [=map/exist=], return None.
```

Comment:
Using map/exists on a list doesn't make sense.

Reply:
Yes. The intent was that this gets called with *configuration["bla"]* and I'm hoisting the existance check into the caller. I don't think that was valid. I've now prepared a version that's explicit.